lstchain/reco/reconstructor.py

```diff
@@ -574,32 +576,33 @@ def log_pdf(self, charge, t_cm, x_cm, y_cm, width,
             it = it + 1
         kmin[kmin<0] = 0
         kmax = np.ceil(kmax)
-        if len(kmin)==0 or len(kmax)==0:
-            kmin,kmax=0,len(self.log_k)
+        mask = (kmax <= len(self.log_k) / 5)  # /5 for testing only
```

Comment:
Is this hardcoded constant related to l511? Please avoid such things. If you need, use a local variable

Reply:
Yes it is the same 5, I can replace it. But it should disappear before merging as it is only used to have a common range for the previous likelihood computation and the new Gaussian approximation.